### ISSUE TITLE
Mountain Duck modification time not updated when using MacVim

### ISSUE THREAD

**Issue by jknockaert** [iterate-ch/cyberduck#15417]
**Describe the bug**
When opening a file that was synced by Mountain Duck using MacVim and making modifications, the modification timestamp does not get updated after saving (`+w` or `+x` in MacVim).

**To Reproduce**
1. Open a connection to a server (using sftp in online connection mode and with index files and buffering disabled)
2. Open a file in MacVim
3. Edit the file and save
4. Modification timestamp not updated nor in the local copy nor on the server

**Expected behavior**
Modification timestamp updated both locally and remotely.

**Desktop (please complete the following information):**
 - OS: macOS 14.1.2
 - MacVim: r178 (Vim 9.0.1897)
 - Version: Mountain Duck 4.15.1 (21679)

**Comment by rwaldspur** [comment]
This behavior is also present when editing a PDF in macOS application "PDF Expert".

Substite "PDF Expoert" for "MacVim" in above description.

OS: macOS v13.6.1
PDFExpert v3.7.1
MountainDuck v4.15.2 (21684)
CPU: Apple M2

**Comment by dkocher** [comment]
> This behavior is also present when editing a PDF in macOS application "PDF Expert", with server side as Google Drive or DropBox.

@rwaldspur Are you working in _Online_ connect mode as well?

**Comment by rwaldspur** [comment]
No, thanks for that clarification.  I am in OFFLINE mode, though I believe I tested online mode as well.

Here are my sync settings.
<img width="578" alt="Mountain Duck Sync settings" src="https://github.com/iterate-ch/cyberduck/assets/29511202/dfe739a1-b3f0-4d49-8bf0-c8e7a0e79cd2">
